hadoop-hdds/docs/content/interface/ReconApi.md

```diff
@@ -232,7 +232,170 @@ Returns the UnhealthyContainerMetadata objects for all the unhealthycontainers.
 Returns the UnhealthyContainerMetadata objects for the containers in the givenstate.
 Possible unhealthy container states are `MISSING`, `MIS_REPLICATED`,`UNDER_REPLICATED`, `OVER_REPLICATED`.
 The response structure is same as `/containers/unhealthy`.
-     
+
+## Namespace Metadata
+
+### GET /api/v1/namespace/summary
+
+**Parameters**
+
+* path
+
+   The path request in string without any protocol prefix.
+
+**Returns**
+
+Returns a basic summary of the path, including entity type and aggregate count 
+of objects under the path.
+
+`status` is `OK` if path exists, `PATH_NOT_FOUND` otherwise.
+
+```json
+     {
+     	"status": OK,
+     	"type": ROOT,
+     	"numVolume": 10,
+        "numBucket": 100,
+        "numDir": 1000,
+        "numKey": 10000
+     }
+```
+
+If any `num` field is `-1`, the path request is not applicable to such an entity type.
+
+### GET /api/v1/namespace/du
+
+**Parameters**
+
+* path
+
+  The path request in string without any protocol prefix.
+
+* files (optional)
+
+  A boolean with a default value of `false`. If set to `true`, compute disk usage for keys 
+  directly under the path.
+
+* replica (optional)
+
+  A boolean with a default value of `false`. If set to `true`, compute disk usage with replicated
+size of keys.
+
+**Returns**
+
+Returns disk usage of all sub-paths under the path. Normalizes `path` fields, returns
+total size of keys directly under the path as `sizeDirectKey`, and returns 
+`size`/`sizeWithReplica` in number of bytes. 
+
+`status` is `OK` if path exists, 
+`PATH_NOT_FOUND` otherwise.
+
```

Comment:
Let's put the example query path & params that gives the result below here as well.